### ISSUE TITLE
Onboarding and updates onboarding

### ISSUE THREAD

**Issue by gingerr** [lwouis/alt-tab-macos#85]
We could throw a panel to inform the user about the Preferences merge. Or it could auto open the PreferencesPanel and show some info alert message on top.

„Your Preferences have been merged to a new version. Some values could have been reverted or be defaults. Please double check all Preferences“ or something like that.

**Comment by lwouis** [comment]
Now that I think about it, I would argue that it's bad UX to always notify the user. If the migration went without any issue, then we should be silent and "just work". We could however put up an alert when some of the previous user preferences don't exist anymore in the new version and got dropped.

However here, I would argue focusing on the preferences that dropped is thinking about the problem space in a narrow way. What we would like to communicate to the user in general is changes between versions. It's an onboarding story. We could have a UI open on first launch after an update showing release notes, or maybe a full blown overlay interactive tutorial or something.

To recap:

* Your proposal: "Hey we migrated the prefs, something may have dropped, please check"
* Disturbing in fewer cases: "Hey these 3 prefs are no longer so we dropped them"
* More holistic onboarding: "The app now automatically does X and Y" (user now understands why their preferences are gone)

What do you think?

**Comment by lwouis** [comment]
I don't think an onboarding is necessary anymore. Not in general for the app, nor for the updates changing things.

It would be nice to onboard a bit more the user, but it's a lot of work for an app that's quite straightforward. The Preferences are a bit too intense currently, but instead of working on adding an onboarding, i would rather work on a better UI for them. Maybe a simple/advanced mode switcher.
